### PR TITLE
Catch installer failure and dont fail the pipeline from it

### PR DESCRIPTION
**Pull request by johnoliver**

### CODE REVIEW

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 1]
```diff
@@ -290,6 +290,60 @@ class Build {
...
+        def installerJob = context.build job: "build-scripts/release/create_installer_mac",
+                propagate: true,
+                parameters: [
+                        context.string(name: 'UPSTREAM_JOB_NUMBER', value: "${env.BUILD_NUMBER}"),
+                        context.string(name: 'UPSTREAM_JOB_NAME', value: "${env.JOB_NAME}"),
+                        context.string(name: 'FILTER', value: "${filter}"),
+                        context.string(name: 'FULL_VERSION', value: "${versionData.semver}"),
+                        context.string(name: 'MAJOR_VERSION', value: "${versionData.major}"),
+                        context.string(name: 'CERTIFICATE', value: "${certificate}"),
+                        ['$class': 'LabelParameterValue', name: 'NODE_LABEL', label: "${TARGET_OS}&&build"],
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy]: comma at the end

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 2]
```diff
@@ -290,6 +290,60 @@ class Build {
...
+                        context.string(name: 'UPSTREAM_JOB_NAME', value: "${env.JOB_NAME}"),
+                        context.string(name: 'FILTER', value: "${filter}"),
+                        context.string(name: 'PRODUCT_MAJOR_VERSION', value: "${versionData.major}"),
+                        context.string(name: 'PRODUCT_MINOR_VERSION', value: "${versionData.minor}"),
+                        context.string(name: 'PRODUCT_MAINTENANCE_VERSION', value: "${versionData.security}"),
+                        context.string(name: 'PRODUCT_PATCH_VERSION', value: "${versionData.build}"),
+                        context.string(name: 'JVM', value: "${VARIANT}"),
+                        context.string(name: 'SIGNING_CERTIFICATE', value: "${certificate}"),
+                        context.string(name: 'ARCH', value: "${ARCHITECTURE}"),
+                        ['$class': 'LabelParameterValue', name: 'NODE_LABEL', label: "${TARGET_OS}&&wix"],
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy]: comma at end

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 3]
```diff
@@ -298,76 +352,24 @@ class Build {
...
-                    context.sh 'for file in $(ls workspace/target/*.zip workspace/target/*.msi); do sha256sum "$file" > $file.sha256.txt ; done'
+        context.node('master') {
+            context.stage("installer") {
+                try {
+                    switch (TARGET_OS) {
+                        case "mac": buildMacInstaller(versionData); break
+                        case "windows": buildWindowsInstaller(versionData); break
+                        default: return; break
+                    }
+                    context.sh 'for file in $(ls workspace/target/*.tar.gz workspace/target/*.pkg workspace/target/*.msi); do sha256sum "$file" > $file.sha256.txt ; done'
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy]: Were we dropping the SHA for PKG files?

Comment (johnoliver) [reply pipelines/build/common/openjdk_build_pipeline.groovy]: not just combined them into 1 command

**Pull request merged**